src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COLUMN_SEPARATOR = ",";
+    private static final int OPERATION_POSITION = 0;
+    private static final int VALUE_POSITION = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileData = readFromFile(fromFileName);
+        String[][] report = createReport(fileData);
+        writeToFile(toFileName, report);
+    }
+
+    private String[][] createReport(String fileData) {
```

Comment:
```suggestion
    private String createReport(String fileData) {
```
createReport should return 'ready to use' report